### DISCUSSION TITLE
Add email back to overview Table Columns

### DISCUSSION THREAD

**Discussion by thomascoppein** [solspace/craft-freeform#1437] [Questions & Answers] [answered]
Hi,

Is there a specific reason why the Email column was removed from the overview options?
This feature seems to have been removed since Freeform 5.

![Screenshot 2024-07-29 at 15 09 28](https://github.com/user-attachments/assets/42de7349-92de-40c0-8c7b-74daba2af09b)

**Comment by kjmartens** [comment, marked as answer]
Hi @thomascoppein,

The list of fields visible as Table columns should display any fields that exist for the form selected. I assume you're not seeing "Email" along with other fields because you are viewing the "All Submissions" tab?

If so, this is a new limitation in Freeform 5 since fields are no longer shared across forms. There isn't a way for Freeform to know which fields are present and similar across all forms, etc.
